cache/carbonlink.go

```diff
@@ -133,13 +133,15 @@ func (listener *CarbonlinkListener) handleConnection(conn net.Conn) {
 
 		reqData, err := ReadCarbonlinkRequest(reader)
 		if err != nil {
+			conn.(*net.TCPConn).SetLinger(0)
```

Comment:
shouldn't it call `conn.Close()` ? if message is malformed there are no sync markers in protocol to find beginning of new message, it all going to be garbage